single_kernel_mongo/utils/mongodb_users.py

```diff
@@ -171,11 +172,18 @@ def get_hosts(self) -> set[str]:
 )
 
 
-CharmUsers = (
+CharmUsernames = {
     OperatorUser.username,
     BackupUser.username,
     MonitorUser.username,
     LogRotateUser.username,
+}
+
+CharmUsers = (
+    OperatorUser,
+    BackupUser,
+    MonitorUser,
+    LogRotateUser,
 )
```

Comment:
I feel we're duplicating `InternalUsers` here with `CharmUsernames`.
Maybe keep only one (I would go for internal users).